.github/workflows/tools.yml

```diff
@@ -21,12 +21,11 @@ jobs:
             subsystem: tools
             label: tools
             run: |
-              NEW_VERSION=$(npm view eslint dist-tags.latest)
-              CURRENT_VERSION=$(node -p "require('./tools/node_modules/eslint/package.json').version")
-              if [ "$NEW_VERSION" != "$CURRENT_VERSION" ]; then
-                echo "NEW_VERSION=$NEW_VERSION" >> $GITHUB_ENV
-                ./tools/dep_updaters/update-eslint.sh
-              fi
+              ./tools/dep_updaters/update-eslint.sh > temp-output
+              cat temp-output
+              MAYBE_VERSION=$(tail -n1 temp-output)
+              if grep -q "NEW_VERSION" <<< "$MAYBE_VERSION"; then $MAYBE_VERSION >> $GITHUB_ENV; fi
+              rm temp-output
```

Comment:
Hey @aduh95 👋🏼 

I finally opted for another approach as the script doesn't exit in case of non-update (exit 0)

I finally take a look at the output of the tail command before setting github env.